skins/laika/src/components/pages/Payment.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div class="column is-full">
+        <form>
```

Comment:
Let's talk about the placement of the Form tag and how we will paginate after the daily.